data/mock-27/dataset-metadata.tsv

```diff
@@ -1,12 +1,12 @@
-name	nifH_even
-citation	
+name	value
+citation	NA
 qiita-id	NA
-raw-data-url-forward-read	
-raw-data-url-reverse-read	
+raw-data-url-forward-read	https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR6666663
```

Comment:
https://s3-us-west-2.amazonaws.com/mockrobiota/latest/mock-27/mock-forward-read.fastq.gz

Reply:
I changed the url from SRA to this one (assuming that's what you meant here)